src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
 // write your code here
+const promise1 = new Promise((resolve, reject) => {
+  const logo = document.querySelector('.logo');
```

Comment:
Ensure that the element with the class 'logo' exists in the DOM to avoid potential errors. Consider adding a check to handle the case where the 'logo' element is not found.